ios/MullvadVPN/Operations/AsyncOperationQueue.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  AsyncOperationQueue.swift
+//  MullvadVPN
+//
+//  Created by pronebird on 30/05/2022.
+//  Copyright © 2022 Mullvad VPN AB. All rights reserved.
+//
+
+import Foundation
+
+class AsyncOperationQueue: OperationQueue {
+    override func addOperation(_ operation: Operation) {
+        if let operation = operation as? AsyncOperation {
+            let categories = operation.conditions
+                .filter { condition in
```

Comment:
I often avoid unnamed arguments for clarity. That's a style preference.

I use `$0` in some cases where I simply return the same thing back like in case of `compactMap { $0 }` but otherwise I usually give arguments their names.

That's not exactly a requirement, but I feel that it makes it easier for others to understand what closure really does at the expense of being a little more verbose.